appcues/src/main/java/com/appcues/data/model/ModelExtensions.kt

```diff
@@ -54,6 +57,11 @@ internal fun AppcuesConfigMap.getConfigPrimitive(key: String): ExperiencePrimiti
     }
 }
 
+internal fun AppcuesConfigMap.getConfigActions(key: String, actionRegistry: ActionRegistry): List<Action> {
+    return getConfig<List<Any>>(key)?.map { MoshiConfiguration.fromAny<ActionResponse>(it) }
+        ?.mapNotNull { response -> response?.toAction(actionRegistry) } ?: arrayListOf()
+}
```

Comment:
This seems fine, just noting that this `ModelExtensions.kt` file has evolved to really just be stuff for `AppcuesConfigMap` used during trait config decoding right? And this one in particular is very specific to a single trait that needs it. It could just be included in the new `TargetInteractionTrait.kt` where it is being used, as an alternative. I don't feel strongly, just made me ponder the organization of this stuff - checking if you have any opinions.

Reply:
maybe renaming to AppcuesConfigMapExt would make more sense, but I think its ok to leave those methods grouped up and available for other traits, even when its very specific they have similar implementation

Reply:
sounds good, yeah agree rename would be fine if you'd like.